[2023-01-08 11:29:40,928] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2023-01-08T11:29:16.452648+00:00 [queued]>
[2023-01-08 11:29:40,934] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2023-01-08T11:29:16.452648+00:00 [queued]>
[2023-01-08 11:29:40,934] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-08 11:29:40,934] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-01-08 11:29:40,934] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-08 11:29:40,945] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): mapping_categorical_vars> on 2023-01-08 11:29:16.452648+00:00
[2023-01-08 11:29:40,948] {standard_task_runner.py:52} INFO - Started process 22979 to run task
[2023-01-08 11:29:40,952] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'mapping_categorical_vars', 'manual__2023-01-08T11:29:16.452648+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/.ipynb_checkpoints/lead_scoring_data_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmp7quumzgp', '--error-file', '/tmp/tmpxz90htcw']
[2023-01-08 11:29:40,953] {standard_task_runner.py:80} INFO - Job 120: Subtask mapping_categorical_vars
[2023-01-08 11:29:40,989] {task_command.py:371} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2023-01-08T11:29:16.452648+00:00 [running]> on host 1f037b111ba5
[2023-01-08 11:29:41,028] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_Scoring_Data_Engineering_Pipeline
AIRFLOW_CTX_TASK_ID=mapping_categorical_vars
AIRFLOW_CTX_EXECUTION_DATE=2023-01-08T11:29:16.452648+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-08T11:29:16.452648+00:00
[2023-01-08 11:29:44,893] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:199 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-01-08 11:29:45,317] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:208 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-01-08 11:29:45,689] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:217 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-01-08 11:29:51,790] {python.py:173} INFO - Done. Returned value was: None
[2023-01-08 11:29:51,796] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=mapping_categorical_vars, execution_date=20230108T112916, start_date=20230108T112940, end_date=20230108T112951
[2023-01-08 11:29:51,835] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-08 11:29:51,867] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
